packages/db/db-client-types.ts

```diff
@@ -330,3 +330,25 @@ export type ListPsaPinRequestResults = {
 export type NameItem = {
   record: definitions['name']['record']
 }
+
+export type UsedStorageItem = {
+  uploaded: string,
+  pinned: string,
```

Comment:
Given `pinned` is an extremly ambitous words ( also uploaded content is pratically pinned on cluster) and it is a common source of confusione, can we be explicit here and use something like:
`psa_requests`
`psa_content`
`psa_pinned`
to avoid any confusion?

It'd be nice to have it renamed across the board. 

If it is easier, given this is not functional issue, this can be done in a follow-up ticket.

Reply:
I agree, its definitely a source of confusion. I feel like it should be done as a separate ticket.